docs/develop/contracts/debug-locally.md

```diff
@@ -0,0 +1,61 @@
+---
+id: debug-locally
+title: Debug Testnet or Mainnet Contract Locally
+sidebar_label: Debug Locally
+---
+
+After contracts are deployed on testnet or mainnet, you might still experience corner case of:
+
+- contract execution logic does not happen in expected way;
+- storage usage is higher than expected;
+- gas cost is higher than expected.
+
+In these cases, you may want to capture the contract state and function call arguments to debug in the local sandbox. You'll make incremental changes, test locally until everything works as expected. You will also likely want to keep the test fixtures to avoid regression in future.
```

Comment:
```suggestion
In these cases you may want to capture the contract state and function call arguments to debug in the local sandbox. You'll make incremental changes and test locally until everything works as expected. You will also likely want to keep the test fixtures to avoid regression in future.
```